endpoints/openrtb2/auction.go

```diff
@@ -910,7 +922,7 @@ func validateUser(user *openrtb.User, aliases map[string]string) error {
 					return errors.New(`request.user.ext.prebid requires a "buyeruids" property with at least one ID defined. If none exist, then request.user.ext.prebid should not be defined.`)
 				}
 				for bidderName := range userExt.Prebid.BuyerUIDs {
-					if _, ok := openrtb_ext.BidderMap[bidderName]; !ok {
+					if _, ok := deps.bidderMap[bidderName]; !ok {
```

Comment:
This is a change from checking against all possible bidders to now comparing against all active bidders. I don't think this is a material change. Hopefully. 🤞